core/Updates/5.0.0-b1.php

```diff
@@ -48,6 +51,14 @@ public function getMigrations(Updater $updater)
         $migrations = $this->getUpdateArchiveIndexMigrations();
 
         $migrations[] = $this->migration->db->addColumns('user_token_auth', ['post_only' => "TINYINT(2) UNSIGNED NOT NULL DEFAULT '0'"]);
+        $migrations[] = $this->migration->db->addColumns('log_conversion', ['pageviews_before' => "SMALLINT UNSIGNED DEFAULT NULL"]);
+
+        $instanceId = SettingsPiwik::getPiwikInstanceId();
+        if (strpos($instanceId, '.matomo.cloud') === false && strpos($instanceId, '.innocraft.cloud') === false) {
```

Comment:
Good call @sgiehl . We could create an issue for the `CLOUD` project. We then put it in the 5.0 release and update our instructions to run this manually after the release. 
The code itself would otherwise work.

Reply:
>if we expect problems on cloud that are reason enough to exclude it, we need to advise our high traffic on premise customers to skip that as well somehow.

The issue on cloud is running the update at the same time for thousands of customers when shared database resources are being used. Based on the query performance tests that have been done, large individual on-premise users shouldn't have a problem.

I've created issue CLOUD-1370 so this can be included in the Matomo 5.0 deployment tasks.